airflow/hooks/presto_hook.py

```diff
@@ -41,6 +61,11 @@ class PrestoHook(DbApiHook):
     conn_name_attr = 'presto_conn_id'
     default_conn_name = 'presto_default'
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.poll_interval = kwargs.pop("poll_interval")
```

Comment:
Is there a reason why you need to use `*args` and `**kwargs` here? It seems to me that normal parameters will also work here.

Reply:
Yes. This can be set from the Connection, however if you want to overwrite the defaults in the constructor you must pull it from kwargs. This is how it's done in other db hooks:
https://github.com/apache/airflow/blob/master/airflow/hooks/mysql_hook.py#L48
https://github.com/apache/airflow/blob/master/airflow/hooks/mssql_hook.py#L36

Would you be more comfortable with specifying the parameters like:
https://github.com/apache/airflow/blob/master/airflow/contrib/hooks/sqoop_hook.py#L57

Reply:
@mik-laj let me know what do you think so I'll continue with the approach which will be accepted.

Reply:
I prefer a list of parameters because it provides better IDE support. args kwargs should only be used if another solution cannot be used. I would be happy if you would add type annotations too.